design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,95 @@
+If not, the operators receive the proper error identifying the step of failure. 
+The first authenticator to utilize this feature is Authn OpenID Connect. 
+
+_For a complete status check flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+
+This doc describes how to expand the status check functionality to other authenticators.
+For each authenticator, a suite of [general](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn_status_general.md) 
+checks are run followed by those specific to the authenticator.  
+
+General checks include:
+
+**NOTE:** Steps that require developer intervention to implement the status check are italicized below
+1. The requesting user has access to the authenticator status webservice
+    1. See the security section [here](https://app.zenhub.com/workspaces/appliance-development-5c9cf04bfa1c204b63eddcc6/issues/cyberark/conjur/1062) for more info about this webservice 
+1. _The authenticator implements the status check_
+1. The account exists
+1. The webservice exists
+1. The authenticator is enabled in the ENV
+1. _Checks specific requirements for the new authenticator_
+
+To properly implement the status check functionality for the new authenticator, you will need to expand on italicized checks 2 and 6. 
+As described in step 2, the status of an authenticator method is checked for implementation. Therefore, you will need to create a new class in the structure of `Authentication::Authn<Type>::Authenticator` and add a status method like so:
+
+**NOTE:** It is important to keep the `Authn<Type>` pattern when defining the class structure. For example, for OIDC, the structure would be `Authentication::AuthnOIDC::Authenticator`.
+
+```ruby
+def status(authenticator_status_input:)
+  Authentication::Authn<Type>::ValidateStatus.new.(
+    # inputs for the authenticator status check
+  )
+end
+```
+
+**NOTE:** This class is already implemented for existing authenticators and therefore, you will only to create add the `status` method for this step
+
+This status method will call the ValidateStatus’s CommandClass and run the new authenticator-specific checks as described in step 6. This new 
+ValidateStatus CommandClass should be in the structure of `Authentication::Authn<Type>::ValidateStatus` and will need to be in a format similar to:
+
+**NOTE:** It is important to keep the `Authn<Type>` pattern when defining the class structure. For example, for OIDC, the structure would be `Authentication::AuthnOIDC::ValidateStatus`.
+
+```ruby
+module Authentication
+  module New_Authenticator
+
+    Err = Errors::Authentication::New_Authenticator
+    # Possible Errors Raised:
+      
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+      },
+      inputs: %i()
+    ) do
+
+def call
+  specific_authn_check1
+  specific_authn_check2
+  .
+  .
+end
+
+private
```

Comment:
please add empty method definitions for `specific_authn_check1 ` & `specific_authn_check2 ` to complete the example